versioned/storage/cache/src/test/java/org/projectnessie/versioned/storage/cache/TestDistributedInvalidations.java

```diff
@@ -43,10 +44,10 @@ public class TestDistributedInvalidations {
 
   protected AtomicLong clockNanos;
 
-  protected CacheBackend backend1noSpy;
-  protected CacheBackend backend2noSpy;
-  protected CacheBackend backend1;
-  protected CacheBackend backend2;
+  CaffeineCacheBackend backend1noSpy;
+  CaffeineCacheBackend backend2noSpy;
+  CaffeineCacheBackend backend1;
+  CaffeineCacheBackend backend2;
```

Comment:
Need the concrete type here, otherwise Mockito complains about the new `cachePut()` being seen as an unwanted interaction, so had to add some `.verify()` for those.